particles/Native/Kotlin/src/wasm/arcs/main.wasm.js

```diff
@@ -0,0 +1,766 @@
+var konan = { libraries: [] };
+
+// NOTE: THIS FILE IS AUTO-GENERATED!
+// Run ':runtime:generateJsMath' to re-generate it.
+
+// NOTE: THIS FILE HAS BEEN HAND MODIFIED (for now) due to it always executing main() on import.
```

Comment:
What were the modifications? Maybe checking in unmodified, then modifying, as separate PRs to make it easier to see would be worthwhile.

I wondered whether having a high level TODO file, README.md or similar to track the biggest known issues, notes, etc. could be worthwhile. To make it easier for folks new to this portion of the code base to get a feel for where things are at.

Reply:
It'll go away in the next update (this will be deleted)